docs/pages/Troubleshooting.md

```diff
@@ -51,4 +51,16 @@ UI5 + Karma:
 
 
 !!! warning
-    The combination of the `UI5_LOG_LVL` environment variable with the `--log-level` CLI parameter might lead to unexpected results; they should be used interchangeably but not together. The CLI parameter takes precedence over the `UI5_LOG_LVL` environment variable.
\ No newline at end of file
+    The combination of the `UI5_LOG_LVL` environment variable with the `--log-level` CLI parameter might lead to unexpected results; they should be used interchangeably but not together. The CLI parameter takes precedence over the `UI5_LOG_LVL` environment variable.
+
+### Changing UI5 Tooling's Home directory
```

Comment:
Should we call it "home" or "data" directory? 🤔

Reply:
Since the variable is called UI5_DATA_DIR, perhaps "data directory" fits better. Or maybe call the environment variable $HOME ?

Reply:
We want to stick to naming the variable `UI5_DATA_DIR` so better to go with "data directory" here 👍

Reply:
I think we decided for 'data' in the var name because `home` often is used for _installation_ directories (JAVA_HOME, MAVEN_HOME, PYTHONHOME). For naming consistency, I therefore would prefer `data` in the documentation, too.

Reply:
Thanks!
I'll update it!